commons/src/main/java/io/hyscale/commons/models/HyscaleSpecType.java

```diff
@@ -17,7 +17,7 @@
 
 public enum HyscaleSpecType {
     SERVICE("service name"),
-    ENVIRONMENT("environment name");
+    PROFILE("profile name");
```

Comment:
should be 
ServiceSpec
ProfileSpec

name does not make sense for HyscaleSpecType right